.circleci/config.yml

```diff
@@ -14,7 +18,8 @@ executors:
   commit-executor:
     docker:
       - image: broadinstitute/study-website-build:04-09-2020A
-    working_directory: ~/repo
+    working_directory: ~/repo/ddp-workspace
+    resource_class: medium+
 
```

Comment:
Made ddp-workspace the default directory. In only one command we need a different working_directory. So removing it from all places where setting it is unnecessary.

Reply:
Also using a bigger instance. (Seems to make very little difference though).